packages/dnb-eufemia/src/elements/typography/style/_dnb-t.scss

```
@@ -0,0 +1,93 @@
+/*
+ * Typography
+ * Universal set of helper classes that do not have a specific element. 
+ * The class ".dnb-t" does nothing, only it's modifiers ".dnb-t__[***]" do.
+ */
+.dnb-t {
+  // size
+  &__size--xx-large {
+    font-size: var(--font-size-xx-large);
+  }
+  &__size--x-large {
+    font-size: var(--font-size-x-large);
+  }
+  &__size--large {
+    font-size: var(--font-size-large);
+  }
+  &__size--medium {
+    font-size: var(--font-size-medium);
+  }
+  &__size--basis {
+    font-size: var(--font-size-basis);
+  }
+  &__size--small {
+    font-size: var(--font-size-small);
+  }
+  &__size--x-small {
+    font-size: var(--font-size-x-small);
+  }
+
+  // line height
+  &__line--xx-large {
```

```suggestion
  &__line-height--xx-large {
```

Not sure if it's necessary, but I feel it makes it a bit easier to read and understand by adding `-height`.

---

I agree it's not optimal, we'll discuss this in tomorrows meeting. 

I think `dnb-t__line--small` works as a CSS class, because you know it's CSS and you know it's typography, but as a prop it's a bit more vague. 

And I would like the prop to match the class, so if we change one we should change both. Though I dislike that we have to use camel case for one and kebab case for the other.